hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -857,10 +870,6 @@ private void initializeDagger(
             initialStateHashFuture = new CompletableFuture<>();
             notifications.register(ReconnectCompleteListener.class, new ReadReconnectStartingStateHash(notifications));
         }
-        if (initialStateHashFuture == null) {
-            logger.warn("Starting from Browser is deprecated. Setting initial start hash to empty hash.");
-            initialStateHashFuture = completedFuture(Bytes.wrap(new byte[48]));
-        }
```

Comment:
How is this related to the network info changes?

Reply:
We had to add all this to support Browser, like added in the log. 
Since I removed supporting Browser in gradle run, removed this